### DISCUSSION TITLE
Clublog upload in Station Location setting disable on its own

### DISCUSSION THREAD

**Discussion by wafer-li** [wavelog/wavelog#938] [Q&A] [answered]
The "Ignore Clublog Upload" option in Station Location setting sometime change to "YES"

I saw the notice

> If this is deactivated on it's own please check if the Call is properly configured at Clublog

But what can I do to prevent it disable clublog uploads

**Comment by HB9HIL** [comment]
You should check if the call (including suffix/prefix) is configured in Clublog and if the password is correct.

**Reply by wafer-li** [reply]
Emm...

I assume the credentials are properly configured because some of the uploads did succeed.

I mean in the same Station Location, the logs was imported into wavelog at the same time via adif.

And after a day or two, most of the log got uploaded into clublog, except for one or two logs did not get uploaded.

When I check the Station Location settings, the clublog upload get disabled on its own.

**Reply by HB9HIL** [reply, marked as answer]
The "Ignore-Setting" gets activated if 
either the response is "Login rejected"
or "Invalid Callsign"

<img width="1308" alt="image" src="https://github.com/user-attachments/assets/7019eb7e-2c8c-4a15-b50a-e07230d656a8">

**Reply by wafer-li** [reply]
I see there is a `log_message` but with "debug"

How can I check the log to deterimine the cause of the problem?

I'm using docker

**Reply by HB9HIL** [reply]
Set the loglevel in your `config.php` to Level 2. You can find your `config.php` in the mounted docker volume.

**Reply by wafer-li** [reply]
I am not sure whether it will happen.

Or just my clublog account is compromised.

I will raise the log level when I import new logs and see whether it will happen again.
